examples/scenarioIntegratorsComparison.py

```diff
@@ -0,0 +1,312 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+r"""
+Overview
+--------
+This scenario illustrates how different integrators compare in terms of
+accuracy and computational cost.
+
+The script is found in the folder ``basilisk/examples`` and executed by using::
+
+      python3 scenarioIntegratorsComparison.py
+
+For information on how to setup different integrators, see :ref:`scenarioIntegrators` and :ref:`scenarioVariableTimeStepIntegrators`. 
+
+Currently, Basilisk only supports explicit Runge-Kutta integrators,
+both of the regular and adaptive variations. Non-adaptive Runge-Kutta
+integrators can be controlled solely by the step size: larger step
+sizes means that faster computation, but less accurate results. 
+
+In contrast, adaptive Runge-Kutta methods are affected both by the step
+size and absolute and relative tolerance. These integrators will try
+to use the user-given step size, but if the error grows too large, a
+smaller time step is used internally for greater accuracy.
+
+It is good practice to use as large a step size as possible for
+adaptive integrators, as this leaves the method greater authority
+to choose the most optimal (internal) step size. However, this means
+that the rest of modules and the results will update with lower
+frequency, which might be undesirable.
```

Comment:
Please add a statement that with Basilisk the largest time step that can be taken is the update period of the Basilisk dynamics module (i.e. like spacecraft).  This is because the module has to output a state message at this update period.

Reply:
What I meant with this paragraph is that increasing the dynamics task time step is good for adaptive integrators, because if they have less constraints about when they need to output state messages, then the adaptive integrator can choose the ideal time step more freely. How about:

```
When using an adaptive integrator, the Basilisk dynamics task time step
can be increased without risk of increasing the integration error. However, 
this also means that other modules in the task are updated less often,
which might be undesirable. Additionally, spacecraft state messages will
also be updated less frequently. 
```

Reply:
I'm good with such a change.  Having the module update rate as a hard limit is a key aspect of using BSK.  I just felt we should be very clear about this limit. I can see users skimming the integrator documentation and missing this.  

Thanks for doing this comparison scenario script. It is a nice study and illustrative to the users.